solr/blacklight-core/conf/solrconfig.xml

```diff
@@ -428,6 +430,7 @@
       
       <!-- for user query terms in number text box -->
       <str name="qf_number">isbn_t</str>
```

Comment:
This is kind of moot because the number search field isn't used
https://github.com/ualbertalib/discovery/blob/af49bd067fcac1e3f5e69114ef789f6dafb0afbd/app/controllers/catalog_controller.rb#L227-L265  and I don't think the advanced section is used at all.
but I think the last declaration would be what is used.  We would probably want them both:
```suggestion
      <str name="qf_number">
        isbn_t
        isbn_cancelled_tim
      </str>
```